app/api/resources/admin.py

```diff
@@ -17,17 +18,38 @@ class AssignNewUserAdmin(Resource):
 
     @classmethod
     @jwt_required()
-    @admin_ns.expect(auth_header_parser, assign_new_admin_request_body, validate=True)
+    @admin_ns.expect(auth_header_parser, assign_and_revoke_user_admin_request_body, validate=True)
     def post(cls):
         """
         Assigns a User as a new Admin.
         """
 
         if current_identity.is_admin:
             data = request.json
-            return DAO.assign_new_user(data)
+            return DAO.assign_new_user(current_identity.id, data)
+
+        else:
+            return {
+                       "message": "You don't have admin status. You can't assign other user as admin."
```

Comment:
403 Forbidden

Reply:
This is fixed @m-murad (it does not show here because you commented on the line above)